crates/topos/tests/snapshots/node__help_display.snap

```diff
@@ -7,9 +7,10 @@ Utility to manage your nodes in the Topos network
 Usage: topos node [OPTIONS] [COMMAND]
 
 Commands:
-  up    Spawn your node!
-  init  Setup your node!
-  help  Print this message or the help of the given subcommand(s)
+  up      Spawn your node!
+  init    Setup your node!
```

Comment:
Total nitpick, but can we drop the exclamation marks? I don't want my CLI tools to yell at me...